cli/src/main.rs

```diff
@@ -947,7 +947,54 @@ fn run<I: IntoIterator<Item = T>, T: Into<OsString> + Clone>(args: I) -> Result<
                                 .help("Database connection URI"),
                         ),
                 ),
-        )
+        );
+
+        app = app.subcommand(
+            SubCommand::with_name("state")
+                .about("Commands to manage scabbard state")
+                .subcommand(
+                    SubCommand::with_name("migrate")
+                        .about(
+                            "Move scabbard state to or from LMDB, deleting from the \
+                            input database",
+                        )
+                        .arg(
+                            Arg::with_name("in")
+                                .long("in")
+                                .help(
+                                    "Database URI that currently contains the scabbard state. If \
+                                    state is in individual LMDB files, provide `lmdb`",
+                                )
+                                .takes_value(true),
+                        )
+                        .arg(
+                            Arg::with_name("out")
+                                .long("out")
+                                .help(
+                                    "The database URI the scabbard state should end up in. \
+                                    If state should be put into individual LMDB files, provide \
+                                    `lmdb`",
+                                )
+                                .takes_value(true),
+                        )
+                        .arg(
+                            Arg::with_name("state_dir")
+                                .long("state-dir")
+                                .help("The location of the state directory for the LMDB files")
```

Comment:
Should we add any info about this defaulting to `SPLINTER_HOME`?

Reply:
added